auth/jwt/middleware.go

```diff
@@ -12,9 +12,9 @@ import (
 type contextKey string
 
 const (
-	// JWTTokenContextKey holds the key used to store a JWT Token in the
+	// JWTContextKey holds the key used to store a JWT in the
 	// context.
-	JWTTokenContextKey contextKey = "JWTToken"
+	JWTContextKey contextKey = "JWT"
```

Comment:
This would be a breaking change in two ways: the API identifier itself has changed, and the actual context key has changed. Let's avoid that.

```suggestion
	JWTContextKey contextKey = "JWTToken"

	// JWTTokenContextKey is an alias for JWTContextKey.
	//
	// Deprecated: exists for historical compatibility only, prefer JWTContextKey.
	JWTTokenContextKey contextKey = JWTContextKey
```

Reply:
I understand. So should I close the issue? or do you have any plan regarding the case in the future?

Reply:
I understand. Upon your suggestion, I revert the value to `JWTToken` and add `JWTTokenContextKey` for historical compatibility.